MODULE_EVALUATION_TEMPLATE.MD

```diff
@@ -15,14 +18,15 @@ When performing a technical evaluation of a module, create a copy of this docume
 * [ ] Module build MUST produce a valid module descriptor
 * [ ] Module descriptor MUST include interface requirements for all consumed APIs
 * [ ] Third party dependencies use an Apache 2.0 compatible license
+* [ ] In order to ensure reproducible builds, snapshot versions of build-time dependencies should not be referenced.
 * [ ] Installation documentation is included
   * -_note: read more at https://github.com/folio-org/mod-search/blob/master/README.md_
 * [ ] Personal data form is completed, accurate, and provided as `PERSONAL_DATA_DISCLOSURE.md` file
 * [ ] Sensitive and environment-specific information is not checked into git repository
 * [ ] Module is written in a language and framework from the [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page
 * [ ] Module only uses FOLIO interfaces already provided by previously accepted modules _e.g. a UI module cannot be accepted that relies on an interface only provided by a back end module that hasn't been accepted yet_
 * [ ] Module gracefully handles the absence of third party systems or related configuration
-* [ ] Sonarqube hasn't identified any security issues, major code smells or excessive (>3%) duplication
+* [ ] Sonarqube hasn't identified any security issues, major code smells or excessive (>3%) duplication; and any disabled rules are reasonably justified by a code comment.
```

Comment:
Same note as above, where I think this should also account for instances where we intentionally ignore sonar violations without disabling the rule.